### ISSUE TITLE
Class Magento\Eav\Model\Entity\Increment\Numeric does not exist (2.0.7)

### ISSUE THREAD

**Issue by sergei-sss** [magento/data-migration-tool#105]
"Class Magento\Eav\Model\Entity\Increment\Numeric does not exist" - error on the save new customer after data migration.

1.9.2.2 -> 2.0.7 (data-migration-tool 2.0.7)
See  vendor/magento/data-migration-tool/etc/ce-to-ce/class-map.xml (313-316)
```xml
    <rename>
        <from>eav/entity_increment_numeric</from>
        <to>Magento\Eav\Model\Entity\Increment\Numeric</to>
    </rename>
```
File Magento\Eav\Model\Entity\Increment\Numeric is missing in 2.0.7

https://github.com/magento/magento2/tree/3f868bc33f81f6294618c295c131567a5da78b6d/app/code/Magento/Eav/Model/Entity/Increment

**Comment by victor-v-rad** [comment]
Hi @sergei-sss 

It was fixed in Data Migration Tool 2.1 . Please try it

**Comment by Puru2016** [comment]
Where I can find the missing file so i can put it on current setup and try to solve this issue? if possible
Or
Should I need to move with latest version of magneto2 and migration tool and follow the migration process again?

**Comment by victor-v-rad** [comment]
>Should I need to move with latest version of magneto2 and migration tool and follow the migration process again?

The fix is in 2.1.0 version. By the way do not forget to save your customization to Data Migration Tool if you have one before updating to 2.1.0

**Comment by Puru2016** [comment]
Thanks for your reply @victor-v-rad , I have fix this issue in 2.0.7 by change some increment_model class in database. Please check here:
http://magento.stackexchange.com/questions/131180/class-magento-eav-model-entity-increment-numeric-does-not-exist-in-magento-2-0-7

**Comment by victor-v-rad** [comment]
The issue was fixed. Please recheck on the latest release.
